hat/test/org/hatdex/hat/api/HATTestContext.scala

```diff
@@ -47,51 +47,155 @@ import org.hatdex.hat.phata.models.MailTokenUser
 import org.hatdex.hat.resourceManagement.{ FakeHatConfiguration, FakeHatServerProvider, HatServer, HatServerProvider }
 import org.hatdex.hat.utils.{ ErrorHandler, HatMailer, LoggingProvider, MockLoggingProvider }
 import org.hatdex.libs.dal.HATPostgresProfile.backend.Database
-import org.specs2.mock.Mockito
-import org.specs2.specification.Scope
+// import org.specs2.mock.Mockito
+// import org.specs2.specification.Scope
 import play.api.cache.AsyncCacheApi
 import play.api.http.HttpErrorHandler
-import play.api.i18n.{Messages, MessagesApi, Lang}
+import play.api.i18n.{ Lang, Messages, MessagesApi }
 import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.{ Application, Configuration, Logger }
 import play.cache.NamedCacheImpl
+import com.dimafeng.testcontainers.{ ForAllTestContainer, PostgreSQLContainer }
 
 import scala.concurrent.duration._
 import scala.concurrent.{ Await, Future }
+import org.scalatestplus.mockito.MockitoSugar
 
-trait HATTestContext extends Scope with Mockito {
+trait HATTestContext extends MockitoSugar {
```

Comment:
I was under the impression that we were using scalamock not Mockito for mocking. Is this going to clash with the scala test common if Mockito is used here ?